src/Gateway.php

```diff
@@ -170,6 +166,13 @@ public function setUsername(string $username): self
         return $this;
     }
 
+    public function setTestMode($testMode): self
+    {
+        $this->setParameter('testMode', $testMode);
+        $this->setBaseUrl();
+        return $this;
```

Comment:
Please add a blank line before return statements ([Symfony CS](https://symfony.com/doc/current/contributing/code/standards.html), I notice this is not defined within the package yet, I'll add it soon)